internal/render/templates/function.tmpl

```diff
@@ -42,11 +42,51 @@ func {{.TestName}}(t *testing.T) {
 		// TODO: Add test cases.
 	}
 	for {{if not .IsNaked}} _, tt := {{end}} range tests {
+		{{if .Subtests}}
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
You should only wrap the code that is different with `if .Subtests`. Otherwise there is too much duplication.